CMakeLists.txt

```diff
@@ -51,13 +51,14 @@ endif()
 # standard dependency macro
 macro(Cabana_add_dependency)
   cmake_parse_arguments(CABANA_DEPENDENCY "" "PACKAGE" "" ${ARGN})
-  find_package( ${CABANA_DEPENDENCY_PACKAGE} QUIET )
   string(TOUPPER "${CABANA_DEPENDENCY_PACKAGE}" CABANA_DEPENDENCY_OPTION )
   option(
     Cabana_REQUIRE_${CABANA_DEPENDENCY_OPTION}
-    "Require Cabana to build with ${CABANA_DEPENDENCY_PACKAGE} support" ${CABANA_DEPENDENCY_PACKAGE}_FOUND)
+    "Require Cabana to build with ${CABANA_DEPENDENCY_PACKAGE} support")
```

Comment:
@junghans Could the `*_FOUND` be removed, as shown here?

Reply:
Then everything would be `OFF` by default.